### ISSUE TITLE
Error when testing different number of layers?

### ISSUE THREAD

**Issue by Aite97** [rstudio/tfruns#60]
Hi, 

love the package, and I have a decent handle on it for tuning most parameters.

However, is it possible to tune how many layers to add in a model?

Say I want to test between 1 and 2 layers in a simple LSTM model example. I tune the units and dropout rate for both layers when included, and always the learning rate.
However, I want the tuning run to test runs where it drops/includes the 2nd layer. 
How can I achieve this in tfruns?

Example of how I tried to do it with strings and a logical statement:

In file where I run the tuning_run() function:
testruns <-tuning_run("lstmmod.R",
                       flags = list(lr = c(0.01, 0.005),
                                    u1 = c(32,64), #units layer 1
                                    u2 = c(16), #units layer 2
                                    d1 = c(0.1), #dropout rate layer 1
                                    d2 = c(0.1), #dropout rate layer 2
                                    exl = c("TRUE","FALSE"))) # In the model file I use an if statement with exl == "TRUE"

In the model file:
FLAGS <- flags(flag_numeric('learn', 0.01),
               flag_numeric('un1',64),
               flag_numeric('un2',32),
               flag_numeric('dr1',0.1),
               flag_numeric('dr2',0.1),
               flag_string('exl', "TRUE"))

# Specify model
model <- keras_model_sequential()
model %>%
  layer_lstm(units=FLAGS$u1, return_sequences = T, dropout = FLAGS$d1)

if(exl == "TRUE") {
  model %>% 
  layer_lstm(units=FLAGS$u2, return_sequences = F, recurrent_dropout = FLAGS$d2)
}

model %>% 
  layer_dense(units=length_sequence) %>%
  compile(loss="mse", optimizer=optimizer_adam(lr = FLAGS$learn))

history <- model %>% fit(x_train, y_train, validation_data=(x_val,y_val), epochs=50)

This failed.

Any help would be appreciated, the key issue is simply how I'd formulate this since the if thing failed.

**Comment by dfalbel** [comment]
You idea is correct! I think instead of `exl` in the if statement you should use `FLAGS$exl` since you are referring to a value in that list.
